python/cac_tripplanner/templates/learn-detail.html

```diff
@@ -4,8 +4,24 @@
 {% block pagetitle %}
 <title>{% block title %}GoPhillyGo | {{ article.title }}{% endblock %}</title>
 
-<meta property="og:title" content="GoPhillyGo | {{ article.title }}" />
-<meta name="twitter:title" content="GoPhillyGo | {{ article.title }}" />
+<meta property="og:title" content="GoPhillyGo | {{ article.title|striptags }}" />
+<meta name="twitter:title" content="GoPhillyGo | {{ article.title|striptags }}" />
+{% endblock %}
+
+{% block extrametatags %}
```

Comment:
It took a surprisingly long time for my brain to parse what this says.
:medal_sports: for me.